plugins/javaembed/javaembed.cpp

```diff
@@ -930,6 +932,23 @@ static class JavaGlobalState
         newPath.append(".");
         optionStrings.append(newPath);
 
+        // Options we should set (but allow for override with jvmoptions below)
+        optionStrings.append("-XX:-UseLargePages");
+
+#ifdef _CONTAINERIZED
+        const char *jvmLibPath = getenv("JAVA_LIBRARY_PATH");
```

Comment:
is this #if code also useful in non-container?

Reply:
Not sure it's worth adding. Environment vars are harder to set in bare-metal than environment.conf values